spec/admin/contributions_controller.spec.rb

```diff
@@ -138,4 +138,50 @@
                   .and have_content(Contribution.human_attribute_name("state/#{contribution_to_show.state}"))
     end 
   end
+
+  describe 'GET edit' do
+    let(:contribution) { create(:contribution) }
+
+    it 'has http status 200' do
+      get :edit, params: { id: contribution.id }
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'assigns the contribution' do
+      get :edit, params: { id: contribution.id }
+
+      is_expected.to render_template(:edit)
+    end
+  end
+
+  describe 'PATCH update' do
+    context 'with valid params' do
+      let(:contribution) { create(:contribution) }
+
+      it "updates contribution's state" do
```

Comment:
```suggestion
      it 'updates contribution's state' do
```

maybe you should use single quotes as in other specs.

Reply:
changed